lib/rubocop/cop/style/format_string.rb

```diff
@@ -53,6 +55,8 @@ class FormatString < Cop
         PATTERN
 
         def on_send(node)
+          return unless FORMAT_METHODS.include?(node.method_name)
```

Comment:
This is a pattern that comes back very often for `on_send`, right? It wouldn't be too hard to tweak `Commissionner` so that a Cop could specify a list of `method_name` to be called for (and not be bothered for others); it would be quite fast. Useful?

Reply:
@marcandre I'd find that incredibly useful for many cops I write.

Reply:
Cool :-) Please check #8365 and let me know how well that fits what you'd like. It could be made more general but I think that `on_send` and `node.method_name` are by far the most important checks.